core/services/workflows/syncer/handler.go

```diff
@@ -0,0 +1,98 @@
+package syncer
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+)
+
+type Handler interface {
```

Comment:
Nit/can follow up: does this need to be public? Same with the interface method?

Reply:
Yea unnecessarily public.